tslint.json

```diff
@@ -1,196 +1,33 @@
 {
-  "rulesDirectory": [
-    "node_modules/codelyzer"
-  ],
+  "extends": "@terminus/tslint-config-frontend/development",
   "rules": {
-    "angular-whitespace": [
-      true,
-      "check-interpolation"
-    ],
-    "arrow-return-shorthand": true,
-    "banana-in-box": true,
-    "binary-expression-operand-order": true,
-    "callable-types": true,
-    "class-name": true,
-    "comment-format": [
-      true,
-      "check-space"
-    ],
-    "component-class-suffix": true,
     "component-selector": [
       true,
       "element",
-      ["ts", "demo"],
+      "ts",
       "kebab-case"
     ],
-    "curly": true,
-    "directive-class-suffix": true,
     "directive-selector": [
       true,
       "attribute",
       "ts",
       "camelCase"
     ],
-    "eofline": true,
-    "forin": true,
-    "import-blacklist": [
-      false
-    ],
-    "import-spacing": true,
-    "indent": [
+    "pipe-prefix": [
       true,
-      "spaces"
+      "ts"
     ],
-    "interface-over-type-literal": true,
-    "jsdoc-format": [
-      true,
-      "check-multiline-start"
-    ],
-    "label-position": true,
-    "max-line-length": [
```

Comment:
These things that were removed here, are they now in eslint?

Reply:
Yep. Now projects should _only_ define their own rules for custom overrides (such as enforcing a custom prefix name for components) or places where they are knowingly diverging from agreed upon conventions.

For instance, the library needs to use console warnings so we allow that specific type through in our own rules, but a codebase like Engage should not.